<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilikarttaRaportti</class>
 <widget class="QWidget" name="TilikarttaRaportti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tulostettavat tilit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="kaikkiRadio">
        <property name="text">
         <string>Kaikki tilit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kaytossaRadio">
        <property name="text">
         <string>Käytössä olevat tilit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kirjauksiaRadio">
        <property name="text">
         <string>Tilit, joilla kirjauksia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="suosikkiRadio">
        <property name="text">
         <string>Suosikkitilit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tilikaudelta</string>
       </property>
       <property name="buddy">
        <cstring>tilikaudeltaCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="tilikaudeltaCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="tilityypitCheck">
       <property name="text">
        <string>Näytä tilityypit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="saldotCheck">
       <property name="text">
        <string>Saldot</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="saldotDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saldotCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>saldotDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
